formal-resume/resume_english.tex

```diff
@@ -75,14 +75,17 @@ \section{Honors}
 \section{Education}
 
 \datedline{\textbf{Changshu Institute of Technology}, China}{Sep 2016 -- Jul 2020}
-  Bachelor of Engineering, Automotive Service Engineering. I also have passed the CET-6 at this period.
+\begin{itemize}
+  \item Bachelors of Engineering, Automotive Service Engineering
+  \item Completed the College English Test at the CET6 level
+  \item Competed in the university's autonomous driving vehicle competition
+\end{itemize}
 
 \section{Skills}
 \begin{itemize}
-  \item I'm not limited to any specific programming language, but I usually use C/C++ and C\#. I can adapt to any tools under Windows and Linux, but I prefer Visual Studio under Windows.
-  \item I have focused on Windows Development with Win32 API, COM (ATL and WTL) and WinRT (C++/WinRT) since 2014, which I started to participate in the Chuyu Team's next generation (at that time) third-party Windows servicing tool which is called DISM++ today.
-  \item I love doing some bare-metal development with Visual Studio under Windows. I had made a Hyper-V guest SDK and a UEFI application SDK to make me happy and achieve my goals.
-  \item I also had the experience for ROS (not ROS2) because I was a member of autonomous driving team when I was a college student.
+  \item I'm not limited to any specific programming language, I am highly proficient in C, C++, and C\#. I am highly experienced in Windows development using Visual Studio, and comfortable in other tooling for Windows and Linux.
+  \item I have focused on Windows Development with Win32 API, COM (ATL and WTL) and WinRT (C++/WinRT) since 2014, which I started to participate in the Chuyu Team's next generation third-party Windows servicing tool, DISM++.
```

Comment:
Consider link to DISM++

Reply:
I will follow the suggestion. But the original official site had down. Only the download link https://github.com/Chuyu-Team/Dism-Multi-language/releases/tag/v10.1.1002.2 is available in the current stage.

Kenji Mouri

Reply:
Done in https://github.com/MouriNaruto/MouriNaruto/commit/5097ef5fe811b10d4d78e2dbb247cff826603b51.